docs/plugins/tv.md

```diff
@@ -1,35 +1,35 @@
 # TV
 
-The TV plugin serves as an abstraction layer to the _live TV_ functionality of a STB. The interface gives you access to the current channel
-and program information, and allows you to change the TV channel from inside an App (if your app is _whitelisted_ to do so).
+The *TV* plugin serves as an abstraction layer to the *live TV* functionality of an STB. The interface gives access to the current channel
+and program information, and allows you to change the TV channel from *inside* an App (if your App is *whitelisted* for that purpose).
 
-With the TV plugin you can adapt your App _contextually_ to what the user is currently watching on TV.
+You can use the TV plugin to adapt your App contextually to what the user is currently watching on TV.
 
 ## Usage
 
-In order to use the TV plugin, import it from the Lightning SDK.
+If you want to use the TV plugin, import it from the Lightning SDK:
 
 ```js
 import { TV } from '@lightningjs/sdk'
 ```
 
 ## Available methods
 
-### Channel
+### channel
 
-Either _retrieves_ information about the TV channel that is _currently_ being watched or _changes_ the current TV channel.
+Either *[retrieves](#retrieve-current-channel)* information about the TV channel that is currently being watched, or *[changes](#change-current-channel)* the current TV channel.
 
-#### Retrieve current Channel
+#### Retrieve Current Channel
 
 ```js
 TV.channel().then(channel => console.log(channel))
 ```
 
-The `channel`-method returns a _promise_ which returns the channel information as an object.
+The `channel` method returns a *Promise* that returns the channel information as an object.
```

Comment:
Maybe we should add that the promise is rejected in case of an error (or if the app is now allowed to retrieve the information)